src/App.ts

```diff
@@ -1540,8 +1547,11 @@ function buildSource<IsEnterpriseInstall extends boolean>(
       // When the app is installed using org-wide deployment, team property will be null
       if (
         typeof bodyAsActionOrOptionsOrViewActionOrShortcut.team !== 'undefined' &&
-        bodyAsActionOrOptionsOrViewActionOrShortcut.team !== null
+        bodyAsActionOrOptionsOrViewActionOrShortcut.team !== null &&
```

Comment:
@srajiang I realized this null check ended up being necessary because the `block_suggestions` schema changed from what it was originally when copying it over from the Deno side. Depending on how we decide to consolidate the existing vs. previous `block_suggestions` structure, we may be able to remove this.

Reply:
So, if we use the bolt-js BlockSuggestions schema, does this become unnecessary?

Reply:
yep yep! those errors were being thrown because in the BlockSuggestions schema from Deno, there was no `enterprise_id` property within the `team` property! I added it in to the hybrid schema this PR is currently using so even if we were to remove it now with the current schema, it would probably not throw an error anymore!

Reply:
Update: this was removed due to the split schemas we're now using for next-gen vs non next-gen Block Suggestions use cases!